build.gradle

```diff
@@ -169,4 +182,3 @@ def getWorkingBranch() {
     println "Working branch: " + workingBranch
```

Comment:
NOTE: Should probably remove line 182 before merging.

Reply:
Removed ✅